backend/composer/models.py

```diff
@@ -269,6 +325,18 @@ def name(self):
             return f"{region_name},{layer_name}"
         else:
             return "Unnamed Entity"
+        
+    @property
+    def ontology_uri(self):
+        if self.simple_entity:
+            return self.simple_entity.ontology_uri
+        elif self.region_layer:
+            layer_uri = self.region_layer.layer.ontology_uri if self.region_layer.layer else ""
+            region_uri = self.region_layer.region.ontology_uri if self.region_layer.region else ""
+            return f"{region_uri},{layer_uri}"
+        else:
+            return None
```

Comment:
comment: @D-GopalKrishna I think this else will never be run (it's either simple or region layer)

Reply:
Yes indeed, already did - https://github.com/MetaCell/sckan-composer/pull/252#issuecomment-2031215692. And merged to this branch as well.